data/error_mail_template_data.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
are you sure that this should be **noupdate="1"**, wouldn't that prevent from editing anything related to this template such as "email_from"

Reply:
No. Noupdate attribute indicates if the values in file must be re-imported when module is updated.